src/core/core.js

```diff
@@ -234,6 +235,12 @@ export default class Core {
     const queryTrigger = this.storage.get(StorageKeys.QUERY_TRIGGER);
     const queryTriggerForApi = this.getQueryTriggerForSearchApi(queryTrigger);
 
+    const queryId = this.storage.get(StorageKeys.QUERY_ID);
+    if (queryId) {
+      const event = new AnalyticsEvent('FOLLOW_UP_QUERY').addOptions({ searcher: Searcher.VERTICAL });
```

Comment:
I think this will also trigger for pagination changes, just checking that that's what we want?

Reply:
also filter changes

Reply:
for talking to product, they want to log this when there's a new queryId. Updated!